src/components/NewRuntimeModal.js

```diff
@@ -285,24 +286,32 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
       })
     }
 
+    const customEnvVars = {
+      WORKSPACE_NAME: name,
+      WORKSPACE_BUCKET: `gs://${bucketName}`,
+      GOOGLE_PROJECT: googleProject
```

Comment:
I think the Leo docker-compose already sets `GOOGLE_PROJECT` but shouldn't hurt to have it here, I think. 
https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/init-resources/jupyter-docker-compose-gce.yaml#L19

Reply:
Yeah, I mainly added it for consistency between runtime and app custom env vars set in the UI.